commands/command_prune.go

```diff
@@ -0,0 +1,460 @@
+package commands
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"sync"
+	"time"
+
+	"github.com/github/git-lfs/vendor/_nuts/github.com/rubyist/tracerx"
+
+	"github.com/github/git-lfs/git"
+	"github.com/github/git-lfs/lfs"
+	"github.com/github/git-lfs/vendor/_nuts/github.com/spf13/cobra"
+)
+
+var (
+	pruneCmd = &cobra.Command{
+		Use:   "prune",
+		Short: "Deletes old LFS files from the local store",
+		Run:   pruneCommand,
+	}
+	pruneDryRunArg      bool
+	pruneVerboseArg     bool
+	pruneVerifyArg      bool
+	pruneDoNotVerifyArg bool
+)
+
+func pruneCommand(cmd *cobra.Command, args []string) {
+
+	// Guts of this must be re-usable from fetch --prune so just parse & dispatch
+	if pruneVerifyArg && pruneDoNotVerifyArg {
```

Comment:
This is the standard `git` pattern for options that can be enabled on the command line, or overridden to be false again e.g. when they're set to true in `gitconfig`. See `--ff` vs `--no-ff`, `--squash` vs `--no-squash` and many others. I prefer to feel like `git` whenever possible.